regression-test/data/nereids_tpcds_shape_sf1000_p0/shape/query14.out

```diff
@@ -4,11 +4,10 @@ PhysicalCteAnchor ( cteId=CTEId#0 )
 --PhysicalCteProducer ( cteId=CTEId#0 )
 ----PhysicalProject
 ------hashJoin[INNER_JOIN] hashCondition=((item.i_brand_id = t.brand_id) and (item.i_category_id = t.category_id) and (item.i_class_id = t.class_id)) otherCondition=() build RFs:RF6 class_id->[i_class_id];RF7 category_id->[i_category_id];RF8 brand_id->[i_brand_id]
+--------PhysicalProject
```

Comment:
Is the plan changed from bucket shuffle to broadcast? Pls make sure it is expected.